README.md

```diff
@@ -338,10 +338,43 @@ You can check "ifconfig -a" to check list of interfaces. Ethernet, WiFi and 6loW
 
         $root@OpenWrt:/# /etc/init.d/network restart
 
+##Writing UBIFS image to Flash
+
+For setting up a tftp server on your development machine, refer **TFTP Boot** section above :-
+
```

Comment:
Add a line above this "Once tftp server is set up, follow the instructions given below :"